src/trio/socket.py

```diff
@@ -68,11 +68,14 @@
 )
 
 if sys.implementation.name == "cpython":
-    from socket import (
-        if_indextoname as if_indextoname,
-        if_nameindex as if_nameindex,
-        if_nametoindex as if_nametoindex,
-    )
+    # Apparently, these functions aren't available on Android with
+    # API < 24 (and possibly other platforms)
+    with _suppress(ImportError):
+        from socket import (
+            if_indextoname as if_indextoname,
+            if_nameindex as if_nameindex,
```

Comment:
Can you only suppress errors for this import? The other two are in API versions before 24.

Reply:
Sure thing, changed in a435dd1